src/main/java/org/opensearch/timeseries/model/TaskState.java

```diff
@@ -9,47 +9,47 @@
  * GitHub history for details.
  */
 
-package org.opensearch.ad.model;
+package org.opensearch.timeseries.model;
 
 import java.util.List;
 
 import com.google.common.collect.ImmutableList;
 
 /**
- * AD task states.
+ * AD and forecasting task states.
  * <ul>
  * <li><code>CREATED</code>:
- *     When user start a historical detector, we will create one task to track the detector
+ *     AD: When user start a historical detector, we will create one task to track the detector
  *     execution and set its state as CREATED
  *
  * <li><code>INIT</code>:
- *     After task created, coordinate node will gather all eligible node’s state and dispatch
+ *     AD: After task created, coordinate node will gather all eligible node’s state and dispatch
  *     task to the worker node with lowest load. When the worker node receives the request,
  *     it will set the task state as INIT immediately, then start to run cold start to train
  *     RCF model. We will track the initialization progress in task.
  *     Init_Progress=ModelUpdates/MinSampleSize
  *
  * <li><code>RUNNING</code>:
- *     If RCF model gets enough data points and passed training, it will start to detect data
+ *     AD: If RCF model gets enough data points and passed training, it will start to detect data
```

Comment:
Are these all `AD` for now until you add the forecasting use case?

Reply:
yes, forecast state are different from AD.